src/api/app/views/webui/project/_basic_info.html.haml

```diff
@@ -11,3 +11,9 @@
       %i No description set
     - else
       = render partial: 'webui/shared/collapsible_text', locals: { text: project.description }
+
+  - if User.possibly_nobody.can_modify?(project) && feature_enabled?(:responsive_ux)
```

Comment:
We have a [Pundit policy for projects](https://github.com/openSUSE/open-build-service/blob/6687073ae4cdd8e1948e501fbc0843e6ac95cf52/src/api/app/policies/project_policy.rb), so we should use it:

```suggestion
  - if policy(project).update? && feature_enabled?(:responsive_ux)
```

Reply:
Ah, I always forget about Pundit... Refactoring!